streamer/src/nonblocking/quic.rs

```diff
@@ -899,15 +795,25 @@ fn handle_chunks(
         }
         trace!("packet batch send error {err:?}");
     } else {
+        if let Some(ppm) = &packet_perf_measure {
+            track_streamer_fetch_packet_performance(core::array::from_ref(ppm), stats);
+        }
+        // stats
```

Comment:
remove these commented out code. remove the stats as well if no longer relevant

Reply:
removed